.github/workflows/code.yml

```diff
@@ -175,8 +175,13 @@ jobs:
     needs: [lint, dist, test]
     if: ${{ github.event_name == 'push' && startsWith(github.ref, 'refs/tags') }}
     runs-on: ubuntu-latest
+    env:
+      PYPI_TOKEN: ${{ secrets.PYPI_TOKEN }}
+      ANACONDA_TOKEN: ${{ secrets.ANACONDA_TOKEN }}
```

Comment:
I don't like putting secrets in the env after the codecov exploit. Can't we use `${{ secrets.PYPI_TOKEN }}` in an `if` clause below?

Reply:
Unfortunately not, I shall change these environment variables to `HAS_PYPI_TOKEN` & `HAS_ANACONDA_TOKEN` as discussed